apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/template/response/Template.java

```diff
@@ -0,0 +1,33 @@
+package gov.cdc.nbs.questionbank.template.response;
+
+import java.time.Instant;
+
+public record Template(
+	Long id,
+	String templateNm,
+	String templateType,
+	String xmlPayload,
+	Integer publishVersionNbr,
+	String formCd,
+	String conditionCd,
+	String busObjType,
+	String datamartNm,
+	String recordStatusCd,
+	Instant recordStatusTime,
+	Instant lastChgTime,
+	Long lastChgUserId,
+	String localId,
+	String descTxt,
+	Character publishIndCd,
+	Instant addTime,
+	Long addUserId,
+	String nndEntityIdentifier,
+	Long parentTemplateUid,
+	String sourceNm,
+	Integer templateVersionNbr,
+	String versionNote	
```

Comment:
same as previous comment -- I would probably limit the returned fields to just whats necessary for the UI

Reply:
@mpeels  same as above.